fabric-rendering-v0/src/main/java/net/fabricmc/fabric/api/client/event/HudRenderCallback.java

```diff
@@ -0,0 +1,20 @@
+package net.fabricmc.fabric.api.client.event;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+
+public interface HudRenderCallback {
```

Comment:
Can you remove the v0 addition? You already added this to v1.

Reply:
Oops I missed this

Reply:
Fixed.